CMake/FindAugeas.cmake

```diff
@@ -0,0 +1,49 @@
+# Finds libaugeas
```

Comment:
I don't think we need this. osquery's `CMakeLibs` does some wacky stuff to attempt to find libraries. If you don't need binaries/compilers/utility scripts then it can do the work of finding your static library for you.

For each `libNAME.a` you want linked:
```cmake
ADD_OSQUERY_LINK_CORE("NAME")
```

Which is already included below.